src/index.ts

```diff
@@ -349,7 +349,7 @@ export default <ExportedHandler<Env>>{
 							},
 							method: "POST",
 							body: JSON.stringify({
-								name: threadName ?? question.substring(0, 17) + "...",
+								name: threadName || question.substring(0, 17) + "...",
```

Comment:
This can't be a nullish check otherwise it creates a thread with an empty name when no thread name is provided (really messes up Discords UI)